lib/cobra_commander/js.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module CobraCommander
+  # Calculates js dependencies
+  class Js
```

Comment:
Why did you break these out?

Reply:
just to make the other file smaller, it contained two other classes inside it which make it a bit difficult to read through it. Also it makes it easier to navigate between these classes now.

Reply:
Those classes are private helpers for the class they were wrapped in. I prefer to keep them scoped there. I have some changes coming up which will make it clearer why. For now, could you put them back? Once my changes are in, we can look at this refactoring in isolation.